index.js

```diff
@@ -0,0 +1,33 @@
+const express = require("express");
+const app = express();
+require("dotenv").config();
+const cors = require("cors");
+const { productRouter } = require("./routes/product.routes");
+const { categoryRouter } = require("./routes/category.routes");
+const { cartRouter } = require("./routes/cart.routes");
+const { wishlistRouter } = require("./routes/wishlist.routes");
+const { errorHandler, pathNotFound } = require("./middleware/errorHandlers");
+const { initialiseDBConnection } = require("./db/db.connect");
+app.use(cors());
+app.use(express.json());
+const PORT = 3000;
+
+app.use("/api/products", productRouter);
+app.use("/api/cart", cartRouter);
+app.use("/api/wishlist", wishlistRouter);
+app.use("/api/categories", categoryRouter);
```

Comment:
Any specific reason for giving the path as "/api/products" for each route , why can't that be just "/products"

Reply:
Nothing specific 
For all the api's we have /api/routes right so similarly I have added